### ISSUE TITLE
expandable panel has broken aria

### ISSUE THREAD

**Issue by bkardell** [bkardell/common-panel#38]
Currently, at least in the directive version, aria-expanded isn't set until the user interacts with it once.  Further, it is swapping aria-expanded and aria-hidden which won't give the right advice when a panel begins life closed (or is closed).

**Comment by LJWatson** [comment]
When a panel is expanded (and can be collapsed), aria-expanded="true" should be set on the paneltitle. This should be done from the outset. When the panel is collapsed, the attribute should be toggled to false. This will ensure that when (keyboard) focus moves to the paneltitle, screen readers correctly announce the state of the panel.

Whereas aria-expanded indicates the state of the panel, aria-hidden changes its state (in the accessibility layer). When aria-hidden="true" the content in question is removed from the browser's accessibility tree. It would therefore be used on the container for the panel content (the bit that collapses).
::
So when aria-expanded="true" is set on the paneltitle, set aria-hidden="false" on the panel content. These would then be reversed when the panel was collapsed.
